modules/oe_whitelabel_search/src/Form/SearchForm.php

```diff
@@ -45,58 +45,34 @@ public static function create(ContainerInterface $container) {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state, array $config = NULL): array {
+    if (empty($config['input']['name'])) {
+      return [];
+    }
+
     $form_state->set('oe_whitelabel_search_config', $config);
-    $input_value = '';
 
-    if (!empty($config['input']['name'])) {
-      $input_value = $this->getRequest()->get($config['input']['name']);
-    }
+    $form['#region'] = $config['form']['region'];
 
     $form['search_input'] = [
-      '#prefix' => '<div class="bcl-search-form__group w-100">',
-      '#suffix' => '</div>',
       '#type' => 'textfield',
       '#title' => $config['input']['label'],
       '#title_display' => 'invisible',
       '#size' => 20,
       '#margin_class' => 'mb-0',
```

Comment:
Yes, is used, because if margin_class is not declared, then the template add its own margin:
![image](https://user-images.githubusercontent.com/6751237/170215219-9ef8cbcb-0598-46d8-a779-11d4e9362f2a.png)

Reply:
Yes, I know of that, the question was is it being printed to the element in the page "if it gets added to any of our search_input(s)?".
I looked into it, found that no, it does not get printed, so it is not used, because we remove that wrapper in form-element--oe-whitelabel-search-form.html.twig, so lets remove it.

Reply:
You're right. Sorry